interp/type.go

```diff
@@ -899,6 +908,8 @@ func isComplete(t *itype, visited map[string]bool) bool {
 	case interfaceT, structT:
 		complete := true
 		for _, f := range t.field {
+			// Fiedld implicit type names must be marked as visited, to break false circles.
+			visited[f.typ.path+"/"+f.typ.name] = true
 			complete = complete && isComplete(f.typ, visited)
```

Comment:
```suggestion
			complete &= isComplete(f.typ, visited)
```

Reply:
No! It's not equivalent.You are replacing a logical `and` where right side is not evaluated if left is false, with an arithmetic bitwise `and` (not boolean), where the operand is called unconditionally.

Reply:
indeed